docs/core/porting/third-party-deps.md

```diff
@@ -42,14 +35,19 @@ netstandard1.3
 netstandard1.4
 netstandard1.5
 netstandard1.6
+netstandard2.0
 netcoreapp1.0
+netcoreapp1.1
+netcoreapp2.0
 portable-net45-win8
 portable-win8-wpa8
 portable-net451-win81
 portable-net45-win8-wpa8-wpa81
 ```
 
-These are the Target Framework Monikers (TFM) which map to versions of the [.NET Standard](../../standard/net-standard.md) and traditional Portable Class Library (PCL) profiles which are compatible with .NET Core.  Note that `netcoreapp1.0`, while compatible, is for applications and not libraries.  Although there's nothing wrong with using a library which is `netcoreapp1.0`-based, that library may not be intended for anything *other* than consumption by other `netcoreapp1.0` applications.
+These values are the [Target Framework Monikers (TFM)](../../standard/frameworks.md) that map to versions of the [.NET Standard](../../standard/net-standard.md), .NET Core, and traditional Portable Class Library (PCL) profiles that are compatible with .NET Core. Note that `netcoreapp`, while compatible, is for .NET Core projects only. Although there's nothing wrong with using a .NET Core library (`netcoreapp`-based), that library can only be consumed by other .NET Core apps.
+
+Alternatively, you can see the TFMs that each package supports on [nuget.org](https://www.nuget.org/) under the **Dependencies** section of the package page.
```

Comment:
> The easiest way to inspect NuGet Package folders is to use the NuGet Package Explorer tool. After downloading it, use the following steps to see the folder names:
> …
> Alternatively, you can see the TFMs that each package supports on nuget.org under the **Dependencies** section of the package page.

How is installing a Windows-only tool and following a 5-step process easier than checking a website? Shouldn't it be the other way around, i.e. talk about nuget.org first and mention NPE as the alternative?

Reply:
I think it's a good idea. I think the issue is that NuGet doesn't always have that info. So instead of easiest, we could say that the tool is the more reliable way to check or something. See the package we use here as an example: https://www.nuget.org/packages/Huitian.PowerCollections/ and you'll see the dependencies are not listed